packages/styles/README.md

```diff
@@ -0,0 +1,3 @@
+# HIG Styles
+
+This is a tentative package for styles and design tokens used in HIG.
```

Comment:
Our designers and external communication refer to atomic-level, or token-level elements as "basics". Things like colors, font-families and weights, spacings—things too small to mean anything on their own. In the spirit of using a ubiquitous language shared by all of our team members and customers, I'd like to use the term "basics" to refer to these elements.

Reply:
@morrisallison strong feelings on this? The way you described it to me, tokens are shared across platforms. When I compare desktop colors vs. web colors, I see different names and values for what is supported, which leads me to think about these particular colors as `Basics`.

Reply:
@eskfung @nfiniteset 

Based on what currently exists as Basics in the project, I view Tokens, Styles, and Basics as separate classifications. The difference is more technical rather than methodical, and stems from the idea that a component may be a Basic, but not all Basics are components.

* Tokens
    - Ubiquitous values that can be used across platforms.
    - .e.g pixel values and hex colors
* Styles
    - Platform specific styles that exist outside the scope of a single component.
    - e.g. `@font-face`, `<body>` styles
* Basics
    - Tokens, styles, and atom components.
    - .e.g hex colors, `Grid` component, `Icon` component, `Image` component

I didn't use the term Basics, because the intention isn't to put any components in this package.

Reply:
I see what you mean here @morrisallison. Basics are a superset. 👍